mindocr/losses/det_loss.py

```diff
@@ -215,22 +216,24 @@ def ohem_batch(self, scores, gt_texts, training_masks):
         return selected_masks
 
     def ohem_single(self, score, gt_text, training_mask):
+        h, w = score.shape[0:2]
+        k = int(h * w)
```

Comment:
Not important in this case because we don't support dynamic shaping, but for the future:
during the graph construction all constants (integers, floats, etc.) will be calculated once and will not be changed anymore. So, need to be careful if you intend `k` to change.

Reply:
Ok, noted.